### ISSUE TITLE
"Initial poll did not succeeed" - Firefox browser extension for Facebook

### ISSUE THREAD

**Issue by jernst** [snarfed/bridgy#1035]
No other information available.

1. Installed Firefox plugin
2. Logged into Facebook
3. Test post
4. Manually triggered "Poll Facebook now" in Firefox about:addons. After a while: "Initial poll did not succeeed"
5. A few minutes later, same message for Instagram also in about:addons. There I did not manually trigger anything.
6. Also, the URL shown https://brid.gy/facebook/upon2020.com 404's ("user was not found"). I'm not clear what the sequence of events is here, so I don't know whether this connected and whether one is the cause of the other, and in which direction.

This is Firefox 88.0.1 on macOS, with the Facebook container extension installed: https://github.com/mozilla/contain-facebook

**Comment by snarfed** [comment]
Hi Johannes! Thanks for trying the extension! And sorry for the trouble. Sadly I think it's still not compatible with the Firefox Container Tabs extension. Mind trying with that one disabled? If so, that's the problem. 😐

**Comment by jernst** [comment]
Nope. First tried by allowing it to "Run in private windows", and then I disabled the Facebook container extension and logged into Facebook from the main window. In each case, after a while after clicking "Poll Facebook now" I'm getting "Initial poll did not succeed".
(I don't know anything about browser plugin. Is there a log or something I can take a look at?)

**Comment by snarfed** [comment]
Thanks for trying! And for offering to send logs, those would definitely be helpful. Here's how to get them:
1. Open `about:debugging`
2. Click _This Firefox_ on the left
3. Scroll down to Bridgy
4. Click _Inspect_ <img width="998" alt="image" src="https://user-images.githubusercontent.com/778068/119147612-9c4d2580-ba00-11eb-8d91-39487a662288.png">
5. Click on the _Console_ tab
6. Right click, _Export Visible Messages To_, _File_, save the file. <img width="391" alt="image" src="https://user-images.githubusercontent.com/778068/119147959-e6360b80-ba00-11eb-8e35-647850177f4c.png">
7. Email the file to bridgy@ryanb.org. (Don't post here because it has tokens and cookies.)

**Comment by snarfed** [comment]
Tentatively closing. Feel free to reopen!

**Comment by timnolte** [comment]
I'm getting this same problem in Google Chrome with the extension.

**Comment by snarfed** [comment]
@timnolte hmm, sorry for the trouble! Does your Facebook profile have your web site in it? That turned out to be the original root cause here. More background: https://brid.gy/about#profile-link , and I'll work on surfacing that error in the UI better.

**Comment by timnolte** [comment]
@snarfed ah, I'll have to check that. Thanks for pointing that out.

**Comment by timnolte** [comment]
@snarfed so on Instagram I do have the "website" set to my website, but added it into my bio field. On Facebook, I didn't have it in either place but have just now added as a website and into my bio.

**Comment by snarfed** [comment]
Great! The next catch is that Bridgy [only handles fully public accounts and posts](https://brid.gy/about#fully+public+posts). Looks like your Instagram account is private, and none of your recent Facebook posts are public, so it's not currently backfeeding any of them.

**Comment by timnolte** [comment]
OK, yeah, I forgot about that aspect. My Facebook profile is public but apparently my posts are being published from Instagram as Friends-only. I've changed some to public but will have to see what I can do about that. I suspect that since my Instagram is private that I can't make the Facebook posts it creates public. I did see the error go away about Facebook after adding my website to my profile. I haven't seen anything come through for the posts I made I changed to public though.
